examples/README.md

```diff
@@ -729,10 +730,20 @@ python /var/tmp/qsim-example.py
 This blueprint provisions a simple cluster for use with a Simcenter StarCCM+
 tutorial.
 
-> **_NOTE:_** The tutorial has not yet been published.
+> The main tutorial is described on the [HPC Toolkit website](https://cloud.google.com/hpc-toolkit/docs/simcenter-star-ccm/run-workload).
 
 [starccm-tutorial.yaml]: ../community/examples/starccm-tutorial.yaml
 
+
```

Comment:
One more flag from the pre-commit checks:
```
PyMarkdown...............................................................Failed
- hook id: pymarkdown
- exit code: 1

examples/README.md:737:1: MD012: Multiple consecutive blank lines [Expected: 1, Actual: 2] (no-multiple-blanks)
```

Reply:
Fixed.